src/figuresClasses.js

```diff
@@ -0,0 +1,80 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.Rectangle = exports.Circle = exports.Triangle = void 0;
+exports.getInfo = getInfo;
+var Triangle = /** @class */ (function () {
+    function Triangle(color, a, b, c) {
+        this.color = color;
+        this.a = a;
+        this.b = b;
+        this.c = c;
+        this.shape = 'triangle';
+        this.area = this.getArea();
+        if (color === undefined ||
+            a === undefined ||
+            b === undefined ||
+            c === undefined) {
+            throw new Error('All values must be entered');
```

Comment:
[CODE STYLE] - If you throw an error try to provide its detailed description in error message. The message 'All values must be entered' could be more descriptive by specifying which values are missing.